src/docs/doc.yaml

```diff
@@ -1,6 +1,8 @@
 paths:
   /api/auth/register:
     post:
+      tags:
+        - register
```

Comment:
I think the tag can be one tag` local authentication` for the register, the login and the logout